.gitignore

```diff
@@ -1,7 +1,9 @@
 !.gitignore
 
+.idea/
```

Comment:
These aren't Python-specific and should go in your personal global gitignore config file.

Reply:
You're right. I remove then entry.

Reply:
@alanhamlett , what's next ?